libs/features/links.js

```diff
@@ -0,0 +1,28 @@
+let fetched = false;
+let linkData = null;
+
+const getLinks = async (path) => {
+  if (!path) return null;
+  if (!fetched) {
```

Comment:
For any given page this will always be the same path right? `fetched` isn't needed.
```suggestion
  if (!linkData) {
```

Reply:
The request might fail, and if it does then the linkdata is not sent, but I also don't want to send a second request. That's why I'm using the fetch variable.